src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -331,7 +385,60 @@
   end
 
   describe '#setup_codemirror_editor' do
-    skip('Please add some tests')
+    it 'sets the style for the editor' do
```

Comment:
Sure, but you're testing the exact implementation here. What if use in the future content_foobar instead of content_for? We need to touch all tests here!
So I still think the content_for belongs to the view and only test the uuid here!

@openSUSE/open-build-service what do you think?

https://www.youtube.com/watch?v=URSWYvyc42M